.github/workflows/docs.yml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: generate-validate-docs
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
```

Comment:
This is going to be hard to maintain. Now, this information is split between code and config.

Reply:
What about changing this to `docs/*` which would be a little less brittle?

Reply:
Yes, that works! Thanks